cmd/gke-exec-auth-plugin/main.go

```diff
@@ -44,12 +44,12 @@ func init() {
 	metav1.AddToGroupVersion(scheme, schema.GroupVersion{Version: "v1"})
 	clientauthv1alpha1.AddToScheme(scheme)
 	clientauthentication.AddToScheme(scheme)
-
-	// Override the default in klog. There's verbosity flag for suppressing output.
-	flag.Set("logtostderr", "true")
 }
 
 func main() {
+	klog.InitFlags(nil)
```

Comment:
looks like client-go is using klog/v2 now... we need to switch klog usage to that

Reply:
https://github.com/kubernetes/cloud-provider-gcp/pull/172